exporter.py

```diff
@@ -211,6 +211,9 @@ def get_odoo_conf(config):
     parser.add_argument('--no-interactive', action='store_true')
     parser.add_argument('--no-attendance', action='store_true')
     parser.add_argument('--select-reviewer', default=False, action='store_true')
+    parser.add_argument('-f', '--auto-submit',
+                        default='ask', type=str,
+                        help='The script will submit the timesheet after import default is "ask". Can be yes, no, or ask')
```

Comment:
```suggestion
    parser.add_argument('-s', '--submit',
                        default='a', type=str,
                        help='The script will submit the timesheet after import default is "ask". Can be yes (y), no (n), or ask (a)')
```
Not sure why we need "ask"... I would always ask for a confirmation. WDYT?

Reply:
The current behaviour is asking confirmation to submit timesheets interactively.

In my workflow, I always double-check in tempo that everything is correct, and submit my timesheets from there. That's why I want to have the 'no' option (in my gtimelogrc) so it doesn't event ask me to submit timesheets.